akka-cluster/src/multi-jvm/scala/akka/cluster/RemoteFeaturesWithClusterSpec.scala

```diff
@@ -27,10 +27,11 @@ class ClusterRemoteFeaturesConfig(artery: Boolean) extends MultiNodeConfig {
   val third = role("third")
 
   private val baseConfig = {
+    val port =  if (artery) MultiNodeSpec.udpPort else MultiNodeSpec.tcpPort
```

Comment:
This tests has implementations for both TCP and UDP. 

When running in multi-node under the aeron-udp build, the variation that test classic remote will still run on TCP. It will disable artery pick the TCP port.